test/JDBC/expected/babel_index_nulls_order-after-15-5-vu-prepare.out

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE babel_index_nulls_order_after_15_5_tbl (a INT, b VARCHAR(10))
```

Comment:
For MVU, views behave little bit differently. It stores parse tree and re-create views during MVU. We need to create views in a prepare script, and test it in a verify script, e.g.,
```
create view ... as select top 1 a ... order by a desc
```